### ISSUE TITLE
[BUG]Conversion of color in string format (starting with #) to RGB is incorrect

### ISSUE THREAD

**Issue by amaztony** [Zingzy/hPyT#38]
Recent code changes infile hPyT.py:
```python
    if isinstance(color, tuple) and len(color) == 3:  # RGB format
        r, g, b = color
        return int(f"{b}{g}{r}", 16)
    elif isinstance(color, str) and color.startswith("#"):  # HEX format
        r, g, b = int(color[1:3], 16), int(color[3:5], 16), int(color[5:7], 16)
        return (r << 16) | (g << 8) | b
    else:
        raise ValueError("Invalid color format. Expected RGB tuple or HEX string.")
```

I changed them as follow:
```python
    if isinstance(color, tuple) and len(color) == 3:  # RGB format
        r, g, b = color
        return (b << 16) | (g << 8) | r
    elif isinstance(color, str) and color.startswith("#"):  # HEX format
        r, g, b = int(color[1:3], 16), int(color[3:5], 16), int(color[5:7], 16)
        return (b << 16) | (g << 8) | r
    else:
        raise ValueError("Invalid color format. Expected RGB tuple or HEX string.")
```

And then it works for me on Windows, in both formats: '#FFFFFF' and (255, 255, 255).

**Comment by Zingzy** [comment]
Hi @amaztony ,
Thank you for catching this and sharing your fix! The order of the color channels was indeed causing incorrect conversions on Windows. Your solution of adjusting the bit shifts for both RGB tuples and HEX strings looks good.
I’d love for you to contribute this fix. I'll assign the issue to you, and if you're up for it, please feel free to open a pull request with your changes. I'd be happy to review and merge it. Your contribution would be greatly appreciated!

Let me know if you need any help with the process.

**Comment by Zingzy** [comment]
Fixed in #40
